.azurePipeline/cibuildwheel_steps.yml

```diff
@@ -0,0 +1,11 @@
+
+steps:
+- task: UsePythonVersion@0
+- bash: |
+    # Use this fork of cibuildwheel for macos c++11 support
+    # https://github.com/joerick/cibuildwheel/pull/156
+    python -m pip install git+https://github.com/Czaki/cibuildwheel
```

Comment:
The template is great, but is this method to install the preferred way to do for Linux?
As it is convenient (instead of doing an `if` on a parameter to provide to the template and all), should we just open an issue locally to modify these lines when the PR you point to is merged in and the code is released?

Reply:
Sure - https://github.com/data61/anonlink/issues/242